cli/src/pcluster/models/cluster.py

```diff
@@ -163,7 +163,9 @@ def _get_cluster_config_dict(self):
         table_name = self.stack.name
         config_version = None
         try:
-            config_version_item = AWSApi.instance().dynamodb.get_item(table_name=table_name, key="CLUSTER_CONFIG")
+            config_version_item = AWSApi.instance().dynamodb_resource.get_item(
```

Comment:
The config version will be read from a CFN parameter as part of a separate PR right?

Reply:
Yes